### PR TITLE
Convert Luminex tests to do non-xar assay setup via remoteAPI

### PR DESCRIPTION
**Pull request by labkey-chrisj**
#### Rationale
The Luminex tests configure an assay with many fields, and doing this via the UI takes time. 
[Issue 41938 ](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=41938) provides details.

#### Related Pull Requests
n/a

#### Changes
This change converts the non-xar assay setup to use the remoteAPI instead of the UI

### CODE REVIEW

File: luminex/test/src/org/labkey/test/tests/luminex/LuminexTest.java [hunk 1]
```diff
@@ -200,64 +204,89 @@ private void doInit()
...
-            resultsPanel.addField("Asymmetry_5pl").setLabel("Asymmetry_5pl").setType(FieldDefinition.ColumnType.Decimal).setNumberFormat("0.0");
-            resultsPanel.addField("Positivity").setLabel("Positivity").setType(FieldDefinition.ColumnType.String);
-
-            assayDesignerPage.clickFinish();
+            Domain resultsDomain = assayProtocol.getDomains().stream().filter(a->a.getName().equals("Data Fields")).findFirst()
+                    .orElseThrow(()-> new IllegalStateException("The protocol template did not supply a [Data Fields] domain"));
+            List<PropertyDescriptor> resultsFields = resultsDomain.getFields();
+            resultsFields.add(new PropertyDescriptor("FIBackgroundNegative", "FI-Bkgd-Neg", "float").setFormat("0.0"));
+            resultsFields.add(new PropertyDescriptor("Standard", "Stnd for Calc", "string"));
+            resultsFields.add(new PropertyDescriptor("EstLogConc_5pl", "Est Log Conc Rumi 5 PL", "float").setFormat("0.0"));
```
Comment (cnathe) [inline luminex/test/src/org/labkey/test/tests/luminex/LuminexTest.java:261]: missed EstConc_5pl

**Review by cnathe — APPROVED** [review]
Excellent. This should be much faster!

One missed property in the results/data domain: EstConc_5pl

File: luminex/test/src/org/labkey/test/tests/luminex/LuminexTest.java (hunk 1 above)
Comment (labkey-chrisj) [reply luminex/test/src/org/labkey/test/tests/luminex/LuminexTest.java:261]: Thanks for the catch!

**Review by labkey-kevink — APPROVED** [review]
Looks good

**Pull request merged**